client/joinus_client/src/api/userEvents.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios"
+
+export default function userEvents() {
+
+  // will be refactoring this into their own files.
+  const userCreateEventSubmit = () => {
```

Comment:
this get request already exists in hooks > useAppData.js

Reply:
Todays' been an odd day - All of these are prototypes, and happy to give you some context: 

The purpose of this function currently is to console log a click from the button popup on the `addEvents.js` to simulate the pathing when the SQL request gets sent to the db: ![Screen Shot 2022-09-30 at 8 22 45 PM](https://user-images.githubusercontent.com/43016199/193384916-718be725-36a9-437a-8693-2bf71d214173.png)

The first mentor I spoke with suggested writing the calls in this directory since he said couldn't talk to the files within `server/*` directories - an error saying it needed to be placed in the `/src` directory was present earlier. The get request is solely there for testing.

The second one I spoke with, suggested writing the SQL calls (after being introduced to postman) within `server/db/queries/events.ts` which is also in the prototyping phase